dash_table/package-info.json

```diff
@@ -39,42 +39,42 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "@babel/cli": "^7.2.0",
-    "@babel/core": "^7.2.0",
-    "@babel/polyfill": "^7.0.0",
-    "@babel/preset-env": "^7.2.0",
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
+    "@babel/polyfill": "^7.2.5",
+    "@babel/preset-env": "^7.2.3",
     "@babel/preset-react": "^7.0.0",
     "@cypress/webpack-preprocessor": "^4.0.2",
     "@percy-io/percy-storybook": "^2.1.0",
-    "@storybook/cli": "^4.0.12",
-    "@storybook/react": "^4.0.12",
-    "@types/ramda": "^0.25.42",
-    "@types/react": "^16.7.13",
+    "@storybook/cli": "^4.1.4",
+    "@storybook/react": "^4.1.4",
+    "@types/ramda": "^0.25.46",
+    "@types/react": "^16.7.18",
     "@types/react-select": "^1.3.4",
-    "babel-loader": "^8.0.4",
-    "core-js": "^2.6.0",
-    "css-loader": "^2.0.0",
-    "cypress": "^3.1.3",
-    "file-loader": "^2.0.0",
+    "babel-loader": "^8.0.5",
+    "core-js": "^2.6.1",
+    "css-loader": "^2.1.0",
+    "cypress": "^3.1.4",
+    "file-loader": "^3.0.1",
     "http-server": "^0.11.1",
     "less": "^3.9.0",
     "less-loader": "^4.1.0",
-    "npm": "^6.4.1",
+    "npm": "^6.5.0",
     "npm-run-all": "^4.1.5",
     "ramda": "^0.26.1",
-    "react": "16.6.3",
+    "react": "16.7.0",
     "react-docgen": "^2.21.0",
-    "react-dom": "16.6.3",
+    "react-dom": "16.7.0",
     "react-select": "^1.3.0",
     "sheetclip": "^0.3.0",
     "style-loader": "^0.23.1",
-    "ts-loader": "^5.3.1",
-    "tslint": "^5.11.0",
+    "ts-loader": "^5.3.3",
+    "tslint": "^5.12.0",
     "typescript": "^3.2.2",
     "wait-on": "^3.2.0",
-    "webpack": "^4.27.1",
-    "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.10",
+    "webpack": "^4.28.3",
+    "webpack-cli": "^3.2.1",
+    "webpack-dev-server": "^3.1.14",
```

Comment:
This is noise from renovate/all -- we don't commit the build result when updating deps